### ISSUE TITLE
feat: live region role support

### ISSUE THREAD

**Issue by jlp-craigmorten** [guidepup/virtual-screen-reader#3]
Handle live region roles:

- alert
- log
- marquee
- status
- timer
- alertdialog

And handle live region attributes:

- aria-atomic
- aria-busy
- aria-live
- aria-relevant

> When live regions are marked as polite, assistive technologies SHOULD announce updates at the next graceful opportunity, such as at the end of speaking the current sentence or when the user pauses typing. When live regions are marked as assertive, assistive technologies SHOULD notify the user immediately.

REF:

- https://w3c.github.io/aria/#live_region_roles
- https://w3c.github.io/aria/#window_roles
- https://w3c.github.io/aria/#attrs_liveregions
- https://w3c.github.io/aria/#aria-live
